scripts/crabdev-deploy/README.md

```diff
@@ -0,0 +1,34 @@
+# Crab Server Installation Scripts
+
+This repo contains scripts to build a CRAB 3 development testbed, assuming you're working on CERN lxplus system and interacting with the Openstack facility.
+
+The typical workflow of the first time you use this script collection is:
+  * `git clone https://github.com/talamoig/crab-testbed # in an area that will be accessible by your VMs later on, eg. on afs`
```

Comment:
Can you add also these scripts to official repo (to this pull) and change documentation?